core/domain/param_domain_test.py

```diff
@@ -28,11 +30,12 @@ class ParameterDomainUnitTests(test_utils.GenericTestBase):
 
     def test_param_spec_validation(self):
         """Test validation of param specs."""
-        param_spec = param_domain.ParamSpec('FakeType')
-        with self.assertRaisesRegex(TypeError, 'is not a valid object class'):
-            param_spec.validate()
 
-        param_spec.obj_type = 'Real'
+        # Ensure that this object class exists.
+        for obj_type in feconf.SUPPORTED_OBJ_TYPES:
+            object_registry.Registry.get_object_class_by_type(obj_type)
```

Comment:
Please separate this into a new test function.

Reply:
added